### DISCUSSION TITLE
Feature Request - add onAsyncLoad prop to async select

### DISCUSSION THREAD

**Discussion by wdfinch** [JedWatson/react-select#4957] [Ideas]
Hi there, so I have an async select compont and I would like to call a function after my `loadOptions` function has been completed. Right now I don't see a way to do this. Therefore I would like to propose adding an `onAsyncLoad` prop that calls a function when the `loadOptions` function has completed.

**Comment by Rall3n** [comment]
Depending on what you want to do, this can already be easily achieved.

There is not much the `Async` component does after calling the `loadOptions` prop besides setting internal states. So you can either run something before you call the callback, or add a `.finally()` handler if you use a `Promise`.
